### PR TITLE
React Capstone

### PR DESCRIPTION
**Pull request by erickhansx**
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have added thorough tests.
- [x] Created components to display the information on the screen. 
- [x] Requested the API to retrieve the Stock data. 
- [x] Created a filter to retrieve stock data based on the country selected. 
- [x] Add the styles to display the app for mobile devices. 
VIDEO: https://www.loom.com/share/9a25981ae12f4002988797016291a5b6
LIVE DEMO: https://zippy-salamander-5d059d.netlify.app/

### CODE REVIEW

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @erickhansx , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has the old ReadMe Template, kindly make sure all main headers from the new [README template](https://github.com/microverseinc/readme-template) are present in your ReadMe.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -1,70 +1,187 @@
...
+If you like this project like and follow.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (Bondok6) [inline README.md]: - [x] Your README is well documented 💯 However, I suggest giving appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 👍

> Requirement ⬇
image

File: README.md [hunk 2]
```diff
@@ -1,70 +1,187 @@
...
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
Comment (Bondok6) [inline README.md]: - [x] I noticed the MIT/LICENSE link is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

- [OPTONAL] Also, I suggest removing this `NOTE ...` recommendation from your README file 👍

File: src/pages/Details.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            </div>
+          </div>
+          <div>
+            <h3>{`Currency: ${detail.currency}`}</h3>
+            <h3>{`Industry: ${detail.industry}`}</h3>
+            <h3>{`Sector: ${detail.sector}`}</h3>
+            <h3>{`CEO: ${detail.ceo}`}</h3>
+          </div>
+        </div>
+      ))}
```
Comment (Bondok6) [inline src/pages/Details.js]: - [x] According to the requirements, You should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including: 
* Typography: font face, size, and weight
* Layout: composition and space between elements.

So Kindly update the design for the details page to look similar to the original design 💯

| Your Design  | Original Design  |   
| --- | --- |
|   |  |

File: src/pages/Companies.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                  <div className="name-market">
+                    <em>{company.companyName}</em>
+                    <em>{`Market Cap: $${(company.marketCap)}`}</em>
+                  </div>
+                </div>
+              </div>
+            </NavLink>
+          </div>
+        ))}
+      </div>
```
Comment (Bondok6) [inline src/pages/Companies.js]: - [x] The same thing here, kindly check my comment above ⬆
> Kindly, update the design for the companies page to look similar to the original design 💯

| Your Design  | Original Design  |   
| --- | --- |
|   |  |

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @erickhansx 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅
- Good video presentation ✅
- The project is deployed properly ✅
- Great job using Redux, Store ✅
- All unit-tests are passed ✅

image



## Required Changes ♻️

- [ ] _Check the comments under the review._ ⬇


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (erickhansx) [reply README.md]: Thank you for the observation, you are right. Is done! : )

File: README.md (hunk 2 above)
Comment (erickhansx) [reply README.md]: Yes, I mistakenly added it to the main branch. I corrected my error. Thanks!

File: src/pages/Details.js (hunk 3 above)
Comment (erickhansx) [reply src/pages/Details.js]: I fixed the layout and fonts, thank you for your feedback!

File: src/pages/Companies.js (hunk 4 above)
Comment (erickhansx) [reply src/pages/Companies.js]: Just did the same! Thank you!

File: src/styles/Company.css [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  flex-direction: column;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* img {
+  max-width: 2em;
+} */
```
Comment (rustam-tolipov) [inline src/styles/Company.css]: **You did a great job @erickhansx.** 

_However, this issue counts as a required change._ 

- [x] To make your code professional, kindly remove unused commented codes from your project :).

File: src/styles/Country.css [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+  display: flex;
+  flex-direction: column;
+}
+
+.example-img {
+  width: 50%;
+}
+
+.country-container > * {
+  /* background-color: #fc5192; */
```
Comment (rustam-tolipov) [inline src/styles/Country.css]: - [x] As mentioned above please remove the unused commented code 😉

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @erickhansx,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
No optional suggestion :)

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/Company.css (hunk 5 above)
Comment (erickhansx) [reply src/styles/Company.css]: Thank you for your feedback. Removed.

File: src/styles/Country.css (hunk 6 above)
Comment (erickhansx) [reply src/styles/Country.css]: Thank you for your feedback. Removed.

File: README.md [hunk 7]
```diff
@@ -1,70 +1,169 @@
...
+- **[Retrieve Stocks from the API]**
+- **[Display the list of countries in the home page]**
+- **[Filter the stocks by country]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (rustam-tolipov) [inline README.md:70]: - [x] It's great you added a Live demo here however I also suggest you add your presentation video section to it.

Comment (erickhansx) [reply README.md:70]: Thank you for your feedback, just added the video to the readme.

Comment (rustam-tolipov) [reply README.md:70]: I like your unit test. It's a great performance. I encourage you to keep up the way you do. Happy coding, and happy weekend.

Comment (erickhansx) [reply README.md:70]: Thank you so much for your kind comments! Happy weekend mate!

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @erickhansx ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**